src/main/java/ch/njol/skript/util/ScriptOptions.java

```diff
@@ -42,14 +42,21 @@
 package ch.njol.skript.util;
 
 import java.io.File;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.Nullable;
 
 /**
  * @author Mirreducki
  * 
  */
 public class ScriptOptions {
```

Comment:
can we just turn this into an all static class instead of a singleton? that is what's done in other similar classes

Reply:
I don't think that should be handled by this pull request, but I could if you'd like